.stickler.yml

```diff
@@ -0,0 +1,7 @@
+linters:
+  rubocop:
+    display_cop_names: true
+  csslint:
+  stylelint:
+  eslint:
+    config: './eslint.config'  
```

Comment:
- [ ] KIndly note that this sticker linter config file is not required for this project. I think it will be better if you remove it 👍🏽

Reply:
Hello @vickymarz i've removed the stickler linter that you mentioned above, please let me know if everything is in order and what my next steps should be